### DISCUSSION TITLE
Implementing <n>g

### DISCUSSION THREAD

**Discussion by hippie68** [helix-editor/helix#10592] [General]
I suggest to implement `<n>g`. The alternatives `<n>gg`, `g<n>g`, and `<n>G` appear unnecessarily cumbersome. I think implementing `<n>g` should be possible, because it seems the only way to use a number with Goto is the simple `Goto line number <n>`.

This handy addition would be in line with the existing `<n>j` and `<n>k`, creating a more consistent experience.

**Comment by pascalkuthe** [comment]
`g` is not a keybinding but a menu. We can't and won't special case it's behavior when there is a count
